app/assets/stylesheets/components/_page_preview.scss

```diff
@@ -0,0 +1,6 @@
+.app-c-preview {
+  .app-c-preview__desktop-preview {
+    width: 100%;
+    height: 1000px;
```

Comment:
I would add `border: 0;` to get rid of the iframe border.

I get `1000px` is an arbitrary number which is ok for now; if we want to not have an inner scroll for the page we'll need some JS on both this component and the inner document/page. We can achieve that with mutation observers, did that for the component preview in the Design System, happy to add it afterwards if we want it do behave this way.